Auto-format buildSrc

pugnascotia opened the pull request:
Opt-in buildSrc for automatic formatting. This required a config tweak
in order to pick up all the Java sources, and as a result more files are
now found in the Enrich plugin, that were previously missed.

@martijnvg would you mind reviewing the handful of changed files for Enrich?

pugnascotia committed:
Auto-format buildSrc

Opt-in buildSrc for automatic formatting. This required a config tweak
in order to pick up all the Java sources, and as a result more files are
now found in the Enrich plugin, that were previously missed.

elasticmachine commented:
Pinging @elastic/es-core-infra (:Core/Infra/Build)

martijnvg commented on buildSrc/src/main/java/org/elasticsearch/gradle/Jdk.java:
```diff
@@ -135,7 +135,8 @@ void finalizeValues() {
         platform.finalizeValue();
-        vendor.finalizeValue();;
+        vendor.finalizeValue();
+        ;
```
maybe just remove this empty statement?

martijnvg reviewed (approved):
Enrich wise LGTM

mark-vieira reviewed (approved):
This all looks pretty reasonable. 👍

pugnascotia committed:
Address review comments

Most notably, replace a number of String concatentations with
String.format().

pugnascotia commented:
@martijnvg thank you, good spot, I've removed the semicolon.

@mark-vieira I moved the two Java files out of the Groovy directory, but I've left the Spotless config as-is because it found some missed-out Java files in the Enrich projet as well. I've also replaced a few String concatentations with `format()` calls, what do you think?

mark-vieira reviewed (approved):
LGTM 👍

pugnascotia committed:
Merge remote-tracking branch 'upstream/master' into format-build-src

pugnascotia commented:
@elasticmachine run elasticsearch-ci/default-distro
